<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dataset</class>
 <widget class="QWidget" name="Dataset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Dataset {
	background: white;
}

QLabel {
	color: red;
	font-weight: bold;
	font-size: 18px;
	height: 35px;
}

QPushButton {
	min-height: 50px;
	max-width: 150px;
	width: 150px;
	font-weight: bold;
	border-radius: 5px;
	color: red;
	background-color: 	#D3D3D3;
}

QPushButton:hover:!pressed {
	background-color: #DCDCDC;
}


QPushButton:pressed, QPushButton[cssClass=&quot;active&quot;] {
	color: white;
	background-color: red;
}

QPushButton:pressed:hover, QPushButton[cssClass=&quot;active&quot;]:hover {
	background-color: #EC3812;
}

QGroupBox {
	font-size: 20px;
	color: red;
	font-weight: bold;
}

QPushButton#delete_button, QPushButton#edit_button {
	max-width: 75px;
}


QListWidget {
	border: 1px solid silver;
	border-radius: 5px;
}

QListWidget#courses_list_widget {
	margin-top: 8px;
	border: 0px;
}

QListWidget::item::selected {
	background-color: red;
	color: white;
}</string>
  </property>
  <property name="active" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="dataset_label">
       <property name="text">
        <string>Dataset: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_dataset_button">
       <property name="text">
        <string>Save as JSON</string>
       </property>
       <property name="cssClass" stdset="0">
        <string>active</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="list_layout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="students_button">
         <property name="text">
          <string>Students</string>
         </property>
         <property name="cssClass" stdset="0">
          <string>active</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="teachers_button">
         <property name="text">
          <string>Teachers</string>
         </property>
         <property name="cssClass" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="courses_button">
         <property name="text">
          <string>Courses</string>
         </property>
         <property name="cssClass" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="list_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="details_groupbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Details</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="name_info_label">
            <property name="text">
             <string>Name: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="id_info_label">
            <property name="text">
             <string>ID:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="courses_list_label">
            <property name="text">
             <string>Courses:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QListWidget" name="courses_list_widget"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="id_input"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="name_input"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QPushButton" name="edit_button">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="delete_button">
              <property name="text">
               <string>Delete</string>
              </property>
              <property name="cssClass" stdset="0">
               <string>active</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
